### PR TITLE
Added a note on how to contribute in the README

### PR DESCRIPTION
**Pull request by phillip-kruger**
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

As discussed at the community meeting, making "How to Contribute" more visible.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -18,6 +18,22 @@
 //
 # MicroProfile OpenTracing
 
+'''
```
Comment (pavolloffay) [inline README.adoc]: Remove this

File: README.adoc [hunk 2]
```diff
@@ -18,6 +18,22 @@
...
+* Fork the repository, make changes, then do a pull request.
+* https://github.com/eclipse/microprofile-opentracing/issues[Create or fix an issues].
+* https://gitter.im/eclipse/microprofile-opentracing[Join us on Gitter to discuss this project].
+* Join our https://calendar.google.com/calendar/embed?src=gbnbc373ga40n0tvbl88nkc3r4%40group.calendar.google.com[bi-weekly meeting] every second Wednesday at https://www.timeanddate.com/time/map/[16h00 GMT]. 
+** https://docs.google.com/document/d/1TbeKThAd9Df6IS0S6lz_YsTmxpFqWRmCL2HjbrAyLPQ/edit#heading=h.t9s5gp6jb2g6[Minutes and Agenda].
+** https://eclipse.zoom.us/j/949859967[Meeting room].
+* Join the discussions on the https://groups.google.com/forum/#!forum/microprofile[MicroProfile Google Group]
+* https://microprofile.io/blog/[Contribute a blog post].
+
+'''
```
Comment (pavolloffay) [inline README.adoc]: remove this

File: README.adoc [hunk 3]
```diff
@@ -18,6 +18,22 @@
...
+
+Do you want to contribute to this project? Here is what you can do:
+
+* Fork the repository, make changes, then do a pull request.
+* https://github.com/eclipse/microprofile-opentracing/issues[Create or fix an issues].
+* https://gitter.im/eclipse/microprofile-opentracing[Join us on Gitter to discuss this project].
+* Join our https://calendar.google.com/calendar/embed?src=gbnbc373ga40n0tvbl88nkc3r4%40group.calendar.google.com[bi-weekly meeting] every second Wednesday at https://www.timeanddate.com/time/map/[16h00 GMT]. 
+** https://docs.google.com/document/d/1TbeKThAd9Df6IS0S6lz_YsTmxpFqWRmCL2HjbrAyLPQ/edit#heading=h.t9s5gp6jb2g6[Minutes and Agenda].
+** https://eclipse.zoom.us/j/949859967[Meeting room].
+* Join the discussions on the https://groups.google.com/forum/#!forum/microprofile[MicroProfile Google Group]
```
Comment (pavolloffay) [inline README.adoc]: Keep the links here but remove duplicated links from section below

File: README.adoc [hunk 4]
```diff
@@ -18,6 +18,22 @@
 //
 # MicroProfile OpenTracing
 
+'''
+== How to contribute
```
Comment (pavolloffay) [inline README.adoc]: Keep the formatting aligned with this file - so use `##`

Move the section before build

File: README.adoc (hunk 2 above)
Comment (phillip-kruger) [reply README.adoc]: Done

File: README.adoc (hunk 3 above)
Comment (phillip-kruger) [reply README.adoc]: Done

File: README.adoc (hunk 4 above)
Comment (phillip-kruger) [reply README.adoc]: Done

**Comment by pavolloffay** [comment]
thanks @phillip-kruger

**Pull request merged**